int8 freshConcreteLevel
bool rotaryError
bool upDownError
